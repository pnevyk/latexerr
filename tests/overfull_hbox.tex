\documentclass{article}

\begin{document}

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed in \({e^{ix} = \cos(x) + i \sin(x)}\) condimentum erat.

\end{document}
