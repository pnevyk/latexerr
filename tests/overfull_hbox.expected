File: overfull_hbox.tex

Warning on line 5: Text after Lorem ip-sum do-lor sit amet, con-secte-tur adip-isc-ing elit. Sed in $[]$ (displayed hyphenated) overflows the line end.
